### DISCUSSION TITLE
Empty combine if your driving your self

### DISCUSSION THREAD

**Discussion by olelaurits** [Courseplay/courseplay#7211] [Q&A] [answered]
Hi am trying to have 1 combine drive with coursplay and then me driving 2 combine at the same field in fs19. Problem is that the coursplay driver with the auger waggon dont want to empty the combine i drive, is that Even possible ? 
please Help!

**Comment by Tensuko** [comment, marked as answer]
That is not possible, read Wiki.
https://github.com/Courseplay/courseplay/wiki/Mode-2-and-Mode-3-FAQ
